doc/website/getting-started/installation.md

```diff
@@ -122,6 +125,14 @@ The `CMakeLists.txt` from `iceoryx_meta` can be used to easily develop iceoryx w
     cmake --build build
     ```
 
+    If the compilation fails it is possible that the compilation failure originates
+    inside the Windows SDK and is not caused by iceoryx, see https://github.com/microsoft/vcpkg/issues/15035#issuecomment-742427969.
+    In this case try to rebuild the cmake files with:
```

Comment:
```suggestion
    In this case, try to rebuild the cmake files with:
```

Reply:
I think your comma suggestion is wrong and: https://writer.com/punctuation-checker/ supports my clain :D

Reply:
Commas in the English languages are confusing for me. I'll ping @ltbj here because I know he cares about it :upside_down_face:

Reply:
The comma is correct for how this sentence is written, and I would keep it as-is. We can invert the sentence to remove the comma, but it's not needed (especially for technical writing).